### PR TITLE
Mobile-first

### PR DESCRIPTION
**Pull request by esmiz-dream**
### Mobile First
In This Pull Request I:
- Added the header of the mobile main page
- Added the headline of the mobile main page
- Added the styling of the above html
- Added linter for HTML and css
- Added the ReadMe file of the project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+                don’t hestiate to contact me.
+            </p>
+            <div class="background-img">
+                <div class="social-media">
+                    <img src="/images/github.png" alt="github">
+                    <img src="/images/linkedin.png" alt="linkedin">
+                    <img src="/images/Vector-2.png" alt="Pinter">
+                    <img src="/images/twiter.png" alt="Twiter">
+                    <img src="/images/m.png" alt="M">
+                </div>
```
Comment (wayungi) [inline index.html:34]: - [ ] Please use a `ul` tag to group icons, and `li` per icon; each icon has a link (`a` tag)
Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.
```
<ul>
   <li><a href="/"><img src="/images/github.png" alt="github"></a></li>
   ...
</ul>
```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+                like what you see and have a project you need coded,
+                don’t hestiate to contact me.
+            </p>
+            <div class="background-img">
+                <div class="social-media">
+                    <img src="/images/github.png" alt="github">
+                    <img src="/images/linkedin.png" alt="linkedin">
+                    <img src="/images/Vector-2.png" alt="Pinter">
+                    <img src="/images/twiter.png" alt="Twiter">
+                    <img src="/images/m.png" alt="M">
```
Comment (wayungi) [inline index.html:33]: - [ ] Please make sure your image paths are correct. Currently, the images are not displaying on the page. Add a `.` before the `/`.
change this `github` to `github`

**_Figma_***
icons1

**_Your design_**
icons2

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @esmiz-dream  

Good job so far!
image


There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [ ] Professional README ✔️ 
- [ ] LInter checks passing ✔️ 
- [ ] Good commit messages ✔️ 





## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KanzaTahreem — CHANGES_REQUESTED** [review]
# Changes Required 🔁

Hi @esmiz-dream 👋

Well done implementing the changes suggested by previous reviewer 👍 Excellent job. **_However, there are still some things that needs to be fixed.  But you are almost there!  💪._**

almost-there

## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

- ✅ Well done so far following the given Figma design
- ✅ Great job with CSS
- ✅ Descriptive PR and README file
- ✅ Correct Linters

However, there is always room for improvement.

## Required Changes ♻

> _Please keep in mind that your long hours of work is appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄_

_Please check the comments under the review._

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Please check the comments under the review._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+                <a class="logo" href="#">EsmiZ</a>
+                <nav class="hum-menu">
+                    <div class="menu-icon"></div>
+                    <div class="menu-icon"></div>
+                    <div class="menu-icon"></div>
+            </header>
+
+            </nav>
+        </section>
+        <section class="headline">
```
[the remark this answers is not in the document]
Comment (esmiz-dream) [reply index.html:48]: Hi @KanzaTahreem
thank you for your constructive comments.
but for this comment, I set the exact font family like as figma, the look is exact, but the linter generate an error like this 
 
image
so I have to add 'sans-serif' generic font, after that it change the look, plc check the source code.

**Review by GedeonTS — APPROVED** [review]
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi @esmiz-dream ,

You are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

 

## Highlights :clap: :tophat: 
- No linter errors :+1: 
- Good README file :+1: 
- Good use of GitHub flow :+1: 
## Optional suggestions :small_airplane: 
- [OPTIONAL] I think it would be better to write more descriptive commit messages, using the imperative form and starting with a capital letter.
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    <main class="main-content">
+        <section class="app-bar">
+            <div class="status-bar">
+                <span>9:41</span>
+                <div class="status">
+                    <div class="network-signal">
+                        <div class="signal first-level"></div>
+                        <div class="signal second-level"></div>
+                        <div class="signal thired-level"></div>
+                        <div class="signal fourth-level"></div>
```
[the remark this answers is not in the document]
Comment (esmiz-dream) [reply index.html:20]: HI @GedeonTS 
thank you bro!
ya the bar is already displayed in the device and initial I didnt included it, but after my first review I commented to make the exact look of figma and I added. I spent more time for this, but honestly, I challenge myself.
thank you!!

**Pull request merged**